### ISSUE TITLE
Set up automated accessibility testing

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/mep-django#153]
<s>need to copy our pa11y-ci config from ppa and get it set up to run in travis.</s>
investigate running lighthouse in CI for this.

**Comment by thatbudakguy** [comment]
it seems like running lighthouse in CI wouldn't get us anything, since:
- it runs aXe under the hood for accessibility
- the other tests it runs, like performance, aren't meaningful if we're running in a travis container

unfortunately, it seems like the [aXe CLI](https://github.com/dequelabs/axe-cli) isn't designed to crawl sites like pa11y-ci is. it seems like we could either:

- just use pa11y and live with potentially missing some issues in ci, since aXe seems better and more widely-used
- write some logic to do the crawling part ourselves and use aXe-CLI (maybe)
- use aXe to test for a11y issues as part of e2e tests, e.g. with Cypress, which we haven't implemented yet but could be nice to have

**Comment by rlskoeser** [comment]
@thatbudakguy thanks for investigating. I think we should use pa11y for now until we have more time to  invest in better tooling and testing.
